package.json

```diff
@@ -52,14 +52,14 @@
     "uuid": "^8.3.2"
   },
   "peerDependencies": {
-    "jest": "^26.6.3 || ^27.0.0",
-    "jest-circus": "^26.6.3 || ^27.0.0",
-    "jest-environment-node": "^26.6.2 || ^27.0.0",
-    "jest-runner": "^26.6.3 || ^27.0.0"
+    "jest": "^26.6.3 || ^27.0.0 || ^28.0.0",
```

Comment:
does this mean after this change we are still compatible with jest 26 and 27?

Reply:
Because if it means we are not compatible with <28 then we need to release it as a major change, otherwise just as a minor version bump.